jax/experimental/jax2tf/tests/correctness_stats.py

```diff
@@ -200,19 +205,35 @@ def _pipewrap(columns):
 
   return '\n'.join(line for line in map(_pipewrap, table))
 
-def prettify_not_yet_implemented() -> str:
-  """Constructs a summary markdown list of the unimplemented primitives."""
-  ordered_unimpl: List[str]
-  ordered_unimpl = sorted(list(map(lambda prim: prim.name, tf_not_yet_impl)))
+def prettify_as_ordered_list(collec: Collection[core.Primitive]) -> str:
+  """Builds an ordered summary markdown list of a collection of primitives."""
+  ordered_list: List[str] = sorted(list(map(lambda prim: prim.name, collec)))
 
   backtick_wrap = lambda prim_name: f'`{prim_name}`'
-  return ', '.join(list(map(backtick_wrap, ordered_unimpl)))
+  return ', '.join(list(map(backtick_wrap, ordered_list)))
 
-def pprint_limitations(limitations: Sequence[Limitation], output_file: str,
-                       template_file: str) -> None:
+prettify_not_yet_implemented = lambda: prettify_as_ordered_list(tf_not_yet_impl)
+
+def prettify_not_yet_covered(covered_set: Set[core.Primitive]) -> str:
+  """
+  Builds an ordered summary markdown list of all the primitives that are
+  implemented but not in the set passed as an argument.
+  """
+  ignore = set([xla.xla_call_p, pxla.xla_pmap_p, pe.remat_call_p, core.call_p,
+                lax.cond_p, lax.scan_p, lax.while_p])
```

Comment:
The "custom_xxx_call_jaxpr" are also well-tested, in the control-flow-ops-test. Perhaps remove them from here. Alternatively (may be even better) run both primitives_tests and control-flows tests when generating the documentation?

Reply:
Addressed in `7725d8db`; we now also run tests in `control_flow_ops_test` and this instruction is added to the template.